noxfile.py

```diff
@@ -92,7 +92,7 @@ def system(session):
     # Install all test dependencies, then install local packages in place.
     session.install("mock", "pytest", "psutil", "google-cloud-testutils")
     session.install("google-cloud-storage")
-    session.install("fastavro")
+    session.install("fastavro==0.24.2")
```

Comment:
Actually, I think we might not need `fastavro` anymore. We switched to Arrow as the serialization format a while ago. What happens if we just remove this line instead?

```suggestion
```

Reply:
yeah seems as though we don't need it. Ran through tests everything seems to be working fine